security/acme-client/src/opnsense/mvc/app/views/OPNsense/AcmeClient/actions.volt

```diff
@@ -48,6 +48,22 @@ POSSIBILITY OF SUCH DAMAGE.
             }
         );
 
+        // hook into on-show event for dialog to extend layout.
+        $('#DialogAction').on('shown.bs.modal', function (e) {
+            $("#action\\.type").change(function(){
+                var service_id = 'table_optional_' + $(this).val();
+                $(".table_optional").hide();
+                if (($("#action\\.type").val() == 'configd') || ($("#action\\.type").val() == 'custom')) {
```

Comment:
probably easier to just $("."+service_id).show() , I don't expect it would mind if the class isn't there.